src/System.Memory/tests/ReadOnlyBuffer/OwnedArray.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+
+namespace System.Memory.Tests
+{
+    internal class OwnedArray : OwnedMemory<byte>
```

Comment:
nit: Can we adjust the existing test concrete OwnedMemory class based off arrays instead of adding a new one?
https://github.com/dotnet/corefx/blob/master/src/System.Memory/tests/Memory/CustomMemoryForTest.cs

We could just add a constructor there that takes in offset/length and add the corresponding private fields.